src/Didomi.ts

```diff
@@ -599,6 +603,28 @@ export const Didomi = {
     }
   },
 
+  /**
+   *  Set user information with Encryption authentication
```

Comment:
if this function accepts both encryption and hashing we'll need to update this comment

Reply:
let me know what you think of the last commit